dev/AppNotifications/WICUtility.cpp

```diff
@@ -0,0 +1,166 @@
+﻿
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "WICUtility.h"
+#include <cwctype>
+#include <winrt/Windows.Foundation.h>
+#include <winrt/base.h>
+#include <externs.h>
+
+#include <ShObjIdl_core.h>
+#include <wincodec.h>
+#include <Ocidl.h>
+#include <windows.h>
+
+typedef enum
+{
+    BMPV_1,
```

Comment:
Please use full names for this.

Reply:
Naming convention will be discussed tomorrow.